config/routes.rb

```diff
@@ -14,6 +14,11 @@
   get("/auth/developer/sign-out", to: "sessions#signout")
   post("/auth/developer/callback", to: "sessions#callback")
 
+  resources :service_updates,
+            only: %i[index],
+            path: "service-updates",
+            controller: "service_updates"
```

Comment:
Not sure if the controller param is necessary if the resource is named the same as the controller 🤷🏻